shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxBody.hs

```diff
@@ -476,6 +482,18 @@ type TransTxOut (c :: Type -> Constraint) era =
     Compactible (Core.Value era)
   )
 
+-- assume Shelley+ type address : payment addr, staking addr (same length as payment), plus 1 word overhead
+instance
+  ( CC.Crypto (Crypto era),
+    HeapWords (CompactForm (Core.Value era))
+  ) =>
+  HeapWords (TxOut era)
+  where
+  heapWords (TxOutCompact _ vl) =
+    3
+      + (1 + 2 * fromIntegral (HS.sizeHash (Proxy :: Proxy (CC.ADDRHASH (Crypto era)))))
```

Comment:
This is the incorrect version from before..

Reply:
i think so, it's different for me locally. let me re-do the changes